osu.Game.Tests/Visual/Multiplayer/TestSceneMultiScreen.cs

```diff
@@ -10,11 +12,15 @@ public class TestSceneMultiScreen : ScreenTestScene
     {
         protected override bool UseOnlineAPI => true;
 
+        [Cached]
+        private MusicController musicController { get; set; } = new MusicController();
```

Comment:
Missing `Add(musicController)`, honestly not sure about caching a local music controller at all, you can just allow permitting null from the multiplayer screen just like `MainMenu`, but it's fine either way I suppose.

Reply:
The direction of https://github.com/ppy/osu/pull/9792 will make this obsolete anyway.